packages/flutter_tools/test/integration.shard/daemon_mode_test.dart

```diff
@@ -27,6 +27,39 @@ void main() {
     daemonProcess.kill();
   });
 
+  testWithoutContext('startup events', () async {
+    final BasicProject project = BasicProject();
+    await project.setUpIn(tempDir);
+
+    const ProcessManager processManager = LocalProcessManager();
+    daemonProcess = await processManager.start(
+      <String>[flutterBin, ...getLocalEngineArguments(), '--show-test-device', 'daemon'],
+      workingDirectory: tempDir.path,
+    );
+
+    final StreamController<String> stdout = StreamController<String>.broadcast();
+    transformToLines(daemonProcess.stdout).listen((String line) => stdout.add(line));
+    final Stream<Map<String, Object?>> stream = stdout
+      .stream
+      .map<Map<String, Object?>?>(parseFlutterResponse)
+      .where((Map<String, Object?>? value) => value != null)
+      .cast<Map<String, Object?>>();
+
+    final [Map<String, Object?> connectedEvent, Map<String, Object?> logMessage] = await Future.wait(<Future<Map<String, Object?>>>[
```

Comment:
I can't believe I can finally say this, but we're going to need to run the formatter on this.

Reply:
... I think we will, anyway? This line seems super long, but none of the checks are failing yet.

Reply:
Ah, I formatted this but noticed the rest of the file was not formatted. I last rebased this _before_ the formatter was enabled.

However, I can't currently sync with master because of https://github.com/flutter/flutter/issues/160704 (I can't run Flutter at all locally), so I'll rebase and fix this up once that is resolved.

Reply:
Looks like I didn't need to do this myself, as this passed here (because formatting wasn't enforced), and then the merge queue applied it to latest code and it passed there anyway :-)